### DISCUSSION TITLE
Help running Himalaya-vim plugin on neovim

### DISCUSSION THREAD

**Discussion by kskarthik** [pimalaya/himalaya#416] [Q&A] [answered]
Hi, I have installed Himalaya-vim plugin via packer, I also installed himalaya cli seperately.

```use 'https://git.sr.ht/~soywod/himalaya-vim'```

When trying to launch himalaya via `:Himalaya`, I get this error. Please help me out with this issue.

Neovim details:

```sh
:~$ nvim --version
NVIM v0.7.2
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by team+vim@tracker.debian.org

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"

Run :checkhealth for more info
```

![image](https://user-images.githubusercontent.com/11899221/195568088-1fddacc5-4970-487d-a08f-db6d0ef22d70.png)

**Comment by soywod** [comment, marked as answer]
Did you try to run `:messages` to see more logs as described in the exception not caught?

**Reply by kskarthik** [reply]
I get this with `:messages`

![image](https://user-images.githubusercontent.com/11899221/195618299-cbe22698-911d-45e9-b2f5-e1b80705fc22.png)

**Reply by soywod** [reply]
It looks like you do not have the latest version of the CLI, can you check `himalaya --version`?

**Reply by kskarthik** [reply]
ooh! I have the older one
```
kar@earth:~$ himalaya  --version
himalaya 0.5.10
```

**Reply by soywod** [reply]
How did you install the CLI?

**Reply by kskarthik** [reply]
After the latest version upgrade & some config tweaks, It works now, Thanks a lot!

**Reply by kskarthik** [reply]
> How did you install the CLI?

Manually from github releases

**Reply by soywod** [reply]
Awesome :slightly_smiling_face:
